### DISCUSSION TITLE
Use of the VersionID

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#514] [Q&A] [answered]
I do not understand in which cases a new ID version (increasing it) of the same notice should be sent.

Reading the documentation:
[https://docs.ted.europa.eu/eforms/latest/schema/notice-information.html#noticeIDSection](url)

And reviewing the eForms shipments that are already being made in production, for example this publication from Belgium:
[https://ted.europa.eu/udl?uri=TED:NOTICE:340523-2023:TEXT:ES:HTML&src=0&tabId=4](url)

We see that the same notice is sent several times but the VersionId is not incremented.

Thanks in advance

**Comment by ferrakaop** [comment, marked as answer]
Version IDs are used before a notice is published.

So if you set the version ID to 01 and all your notices are successfully submitted, they will all have 01 as version ID on TED.

If your submitted notice fails to validate several times, you can increment your version ID and re-submit it again with the same notice ID (you are also free to generate a new notice ID if you want).

Once a notice has been published, the notice ID cannot be used again.  When you submit a Change notice, this new notice will have a different notice ID to the one it is changing.  So a sequence of changes will have separate notices with different notice IDs and most probably will all have version ID 01 if they all validated first time when submitted, which is what you observe in the example you found.

There are more details in the notice lifecycle presentation of the February workshop (though you might need to look quite carefully to see how version ID is used): 
https://op.europa.eu/en/web/ted-eforms/agenda
